FMDB.podspec

```diff
@@ -26,24 +31,28 @@ Pod::Spec.new do |s|
   s.subspec 'standalone' do |ss|
     ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
     ss.dependency 'sqlite3'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-  end
-
-  # build with FTS support and custom FTS tokenizer source files
+    ss.dependency 'FMDB/common'
+	end
```

Comment:
Agreed that this is better.  I'd just kept the separate subspec to minimize changes from the existing podfile.